src/modules/session/thunks.ts

```diff
@@ -52,12 +51,6 @@ export const updateInitialProfile = createAsyncThunk(
 export const getInitialData = createAsyncThunk(
   "session/getInitialData",
   async (_, { dispatch }) => {
-    const songsResponse = await dispatch(songApi.endpoints.getSongs.initiate());
-
-    if ("error" in songsResponse) {
-      return;
-    }
-
```

Comment:
Removing this as there is no need to request new songs every time we get initial data. Songs should only be requested when the user is on the library page.

Reply:
getInitialData gets called on every single page load and we dont need to do that. We can keep it to fetch only on the library where we actually use the data